core-ui/src/components/Predefined/List/PersistentVolumes.list.js

```diff
@@ -14,23 +15,35 @@ import { PersistentVolumesCreate } from '../Create/PersistentVolumes/PersistentV
 const PersistentVolumesList = props => {
   const { t } = useTranslation();
 
+  const { data: storageClasses } = useGetList()(
+    '/apis/storage.k8s.io/v1/storageclasses',
+  );
+
+  const { data: persistentVolumeClaims } = useGetList()(
```

Comment:
We might want to double-check it, but AFAIK, if the bound PVC doesn't exist anymore, the PV has a `released` status. If that's true, then we don't need to make another call to get a list of PVC, just check the status (SC are not the same story, unfortunately).